src/gui/main.cpp

```diff
@@ -114,25 +116,26 @@ CommandLineOptions parseOptions(const QStringList &arguments)
         return option;
     };
 
-    auto showSettingsLegacyOption = QCommandLineOption{{QStringLiteral("showsettings")}, QStringLiteral("Hidden legacy option")};
+    auto showSettingsLegacyOption = QCommandLineOption{{QStringLiteral("showsettings")}, QApplication::translate("CommandLine", "Hidden legacy option")};
```

Comment:
As it is hidden I don't think we need to translate it

Reply:
done